xsl/xccdf-report-oval-details.xsl

```diff
@@ -63,6 +65,30 @@ Authors:
 </xsl:template>
 
 <xsl:template mode='brief' match='ovalres:definition'>
+    <xsl:variable name="definition" select="key('ovaldef-definition', ./@definition_id)" />
```

Comment:
@jan-cerny Please submit this change as a separate PR, it isn't mandatory part of the fix.